### PR TITLE
Clear crops now also clears animated files/crops.

### PR DESCRIPTION
**Pull request by mparent61**
Betty never cleared these before, adding this for completeness before we start bumping up CDN cache times.

@benghaziboy

### CODE REVIEW

File: betty/cropper/models.py [hunk 1]
```diff
@@ -324,11 +324,15 @@ def clear_crops(self, ratios=None):
                 paths += [self.get_absolute_url(ratio=ratio_slug, width=width, format=format)
                           for format in ["png", "jpg"]
                           for width in settings.BETTY_WIDTHS]
+            if self.animated:
+                for format in ['gif', 'jpg']:
```
Comment (benghaziboy) [inline betty/cropper/models.py]: should these kinds of lists be configurable? or at the very leasts constants set in the module?

File: betty/cropper/models.py [hunk 2]
```diff
@@ -446,6 +450,12 @@ def get_absolute_url(self, ratio="original", width=600, format="jpg"):
             "extension": format
         })
 
+    def get_animated_url(self, format="gif"):
+        return reverse("betty.cropper.views.animated", kwargs={
+            "id": self.id_string,
+            "extension": format
```
Comment (benghaziboy) [inline betty/cropper/models.py]: I realize this has been around for sometime now, but we should probably not use `format` since it is a built-in function

**Comment by benghaziboy** [comment]
outside of those 2 comments 👍

File: betty/cropper/models.py (hunk 2 above)
Comment (mparent61) [inline betty/cropper/models.py]: Agree, I'll clean it up

**Comment by mparent61** [comment]
@benghaziboy  updated per both comments, thanks

**Pull request merged**